tools/types/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@airswap/types",
-  "version": "4.0.4",
+  "version": "4.1.0",
```

Comment:
We're still iterating on the minor version (and building the only clients using it so far) so let's just bump this to `4.0.5`.

Reply:
done